app/views/publishers/policy_agreements/new.html.slim

```diff
@@ -0,0 +1,18 @@
+.single-panel--wrapper
+  = render "panel_flash_messages"
+  .single-panel--content
+    .single-panel--padded-content
+      h4.single-panel--headline= t ".heading_html"
+      .text-left
+        = form_for PolicyAgreement.new, { method: :post, url: publisher_policy_agreements_path } do |f|
+          = f.check_box(:accepted_publisher_tos, {}, 'true', 'false')
+          = t('.user_reviewed_tos')
+          = link_to t('.terms_of_service'), "https://basicattentiontoken.org/publisher-terms-of-service/"
+          br
+          br
+          = f.check_box(:accepted_publisher_privacy_policy, {}, 'true', 'false')
+          = t('.user_reviewed_privacy_policy')
+          = link_to t('.privacy_policy'), "https://brave.com/publishers-creators-privacy/"
+          br
+          center = f.submit("Proceed", class: "btn btn-primary")
```

Comment:
I think the `center` HTML element is deprecated, and we should be using something from bootstrap to vertically center if we need to.  Also can we make "Proceed" internationalized :)